File: WebContent/js/service/seguridadService.js
'use strict';

app.service('seguridadService',
    function seguridadService($resource) {
        return $resource('services/seguridadServiceResource/:nbModulo/:nbGrupoModulo/:nbSubGrupoModulo/:nbActividad/:nbHorario', {
        	nbModulo: '@nbModulo',nbGrupoModulo: '@nbGrupoModulo',nbSubGrupoModulo: '@nbSubGrupoModulo',nbActividad:'@nbActividad',nbHorario:'@nbHorario'		
        }, {
            query: {
                method: 'GET',
                isArray: false
            } 
        });
    });


 

 

File: src/ve/org/bcv/services/SeguridadService.java
/**
 * 
 */
package ve.org.bcv.services;

import javax.ejb.Local;

import ve.org.bcv.dto.SeguridadDto;

/**
 * @author Ing Simon Alberto Rodriguez Pacheco
 * 04/08/2016 15:25:16
 * 2016
 * mail : oraclefedora@gmail.com
 */
@Local
public interface SeguridadService {
	SeguridadDto find(String nbModulo,String nbGrupoModulo,String nbSubGrupoModulo,String nbActividad,String nbHorario); 
}
